sims/verilator/verilator.mk

```diff
@@ -6,7 +6,7 @@
 # verilator version, binary, and path
 #########################################################################################
 VERILATOR_VERSION      = 4.016
-VERILATOR_INSTALL_DIR ?= verilator
+VERILATOR_INSTALL_DIR ?= verilator_install
```

Comment:
Renamed since the directory this resides in is called `verilator` so the path would be IMO a bad looking path `verilator/verilator/`